com.chartboost.mediation/Android~/chartboost-mediation-android-bridge/src/main/java/com/chartboost/mediation/unity/AdStore.kt

```diff
@@ -38,9 +39,24 @@ class AdStore {
             fullscreenAdStore.remove(hashCode)
         }
 
+        @JvmStatic
+        fun trackBannerAd(bannerAd: BannerAdWrapper)
+        {
+            val hashCode = bannerAd.hashCode()
+            bannerAdStore[hashCode] = bannerAd;
+        }
+
+        @JvmStatic
+        fun releaseBannerAd(hashCode: Int)
+        {
+            val bannerAd = bannerAdStore[hashCode]
```

Comment:
Do you need to assign this to bannerAd? If so, you can just do

```kotlin
val bannerAd = bannerAdStore.remove(hashCode)
```

If not, just have: 
```kotlin
bannerAdStore.remove(hashCode)
```

Reply:
Done 👍